### ISSUE TITLE
Vuze

### ISSUE THREAD

**Issue by Freethefire** [erickok/transdroid#234]
I hope that this question isn't a duplicate as well! 
I'm guessing that vuze is working as well as it can considering the age of the xml over http plugin. 
Is there no way to use one of the other web uis to connect? The old method is unstable so say the least and it takes a really long time to connect. I've tried with jre7 and 8 so far. 

Best regards 
Alex

**Comment by erickok** [comment]
Yes the old XML over HTTP plugin is deated and not compatible with many (new) jre etc... However, the (I think what is now official) web UI of Vuze is basically a port of Transmission and I have heard that it also supports the full rpc spec, which means you can just connect to the Vuze instance by using Transmission as client type in Transdroid. I haven't tested it myself yet, but heard different users with positive experiences.

**Comment by Freethefire** [comment]
Thanks Eric! Will try asap. Knew that I had seen that port number somewhere before :)

**Comment by erickok** [comment]
Which web UI did you use? I want to add this to the site documentation. Or is it the default one included in Vuze?

**Comment by Freethefire** [comment]
Vuze web remote version 0.5.9 (plugin download from sourceforge) 
Haven't tried it with https yet though, but I think it should work.

**Comment by Freethefire** [comment]
Update:

When adding a torrent I get a error message quite often, it still works well though. Just a bit slower to show the added torrent in the list. Thought I'd mention it for your guide.
